Index.html

```diff
@@ -75,13 +75,62 @@
                         <input type="text" placeholder="Search GitHub">
                     </li>
                     <li><a href="#">Sign in</a></li>
-                    <li><button class="button"><a href="#">Sign up</a></button></li>
+                    <li><button class="button-one"><a href="#">Sign up</a></button></li>
                 </ul>
                 <!-- #endregion -->
             </nav>
-
         </div>
     </header>
+    <section class="section-one">
+        <div class="text-div-one">
+            <h1>Built for developers</h1></br>
+            <p class="grey-text">GitHub is a development platform inspired by the way you work. From <span
+                    class="underline"><a><u>open source</u></a></span> to <span class="underline"><a><u>business</u></a></span>, you can host and review code, manage projects, and build software alongside 50 million developers.</p>
+        </div>
+        <div class="form-div">
+            <form>
+                <label class="form-label" for="Username">Username</label></br>
+                <input class="form" type="text" id="username" name="username"></br>
```

Comment:
The input tags are not closed, with a single element link input or img you need to end it with a trailing slash like so. `<input type="text" />`